modules/admin-ui/src/main/webapp/scripts/modules/events/controllers/serieController.js

```diff
@@ -22,11 +22,20 @@
 
 // Controller for all single series screens.
 angular.module('adminNg.controllers')
-.controller('SerieCtrl', ['$scope', 'SeriesMetadataResource', 'SeriesEventsResource', 'SeriesAccessResource',
-  'SeriesThemeResource', 'ResourcesListResource', 'UserRolesResource', 'Notifications', 'AuthService',
-  'StatisticsReusable',
-  function ($scope, SeriesMetadataResource, SeriesEventsResource, SeriesAccessResource, SeriesThemeResource,
-    ResourcesListResource, UserRolesResource, Notifications, AuthService, StatisticsReusable) {
+.controller('SerieCtrl', ['$scope', 'SeriesMetadataResource',
+  'SeriesEventsResource', 'SeriesAccessResource', 'SeriesThemeResource',
+  'ResourcesListResource', 'UserRolesResource', 'Notifications', 'AuthService',
+  'StatisticsReusable','$http',
+  function ($scope, SeriesMetadataResource, SeriesEventsResource,
+    SeriesAccessResource, SeriesThemeResource, ResourcesListResource,
+    UserRolesResource, Notifications, AuthService, StatisticsReusable, $http) {
+
+    $http.get('/admin-ng/feeds/feeds')
+    .then( function(response) {
+      $scope.feedContent = response.data;
+    }, function(error) {
```

Comment:
iirc, the new(er) AngularJS requires:

```suggestion
    }).catch(function(error) {
```